gobblin-modules/gobblin-kafka-08/src/main/java/org/apache/gobblin/kafka/writer/Kafka08DataWriter.java

```diff
@@ -47,7 +49,7 @@
  *
  */
 @Slf4j
-public class Kafka08DataWriter<D> implements AsyncDataWriter<D> {
+public class Kafka08DataWriter<K,D> implements AsyncDataWriter<D> {
```

Comment:
Should this be <K, V> to match `Kafka09DataWriter`?

Reply:
Ack